File: palindrome_number/palindrome_num.rb
module PalindromeNum
  def self.start
    num = 11
    loop do
      break if num.to_s == num.to_s.reverse && num.to_s(2) == num.to_s(2).reverse && num.to_s(8) == num.to_s(8).reverse
          num += 2
    end
    puts num
  end
end

PalindromeNum.start

File: palindrome_number/palindrome_num.py
num = 11
while True:
    c = str(num)
    b = str(bin(num))[2:]
    o = str(oct(num))[1:]
    if c == c[::-1] and b == b[::-1] and o == o[::-1]:
        break
    num += 2

print num